src/views/Download.vue

```diff
@@ -74,21 +74,58 @@
           <h2>Having trouble installing?</h2>
           <ul>
             <li>Make sure to check your console for any errors - especially during start up</li>
-            <li>Check the <a href="https://github.com/lucko/LuckPerms/wiki" target="_blank">wiki</a> to see if you missed any important steps during setup</li>
+            <li>Check the <a href="/wiki" target="_blank">wiki</a> to see if you missed any important steps during setup</li>
             <li>Delete the <code>libs</code> folder and restart the server to let it regenerate, sometimes this may fix the problem</li>
             <li>If all else fails, join our <a href="https://discord.gg/luckperms" target="_blank">Discord</a> to get some support</li>
           </ul>
         </div>
       </section>
-      <section class="hero">
-        <h1>Extensions</h1>
-      </section>
-      <div class="container">
-        <div class="resources">
-          <div>
-            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolor doloribus eos facilis fuga harum id illum iste, laboriosam molestiae nam necessitatibus non obcaecati quas repellat temporibus veritatis vitae, voluptates voluptatum.</p>
-          </div>
-          <div></div>
+    </div>
+    <section class="hero">
+      <h1>Extensions</h1>
+    </section>
+    <div class="container" style="margin-bottom: -80px;">
+      <div class="resources">
+        <div style="width: 100%;">
+          <p style="text-align: center; font-size: 24px;">Extensions can modify the behaviour of LuckPerms, you can read more about them <a href="/wiki/Extensions">on the wiki</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="container">
+      <div class="resources">
+        <div style="width: auto; padding-top: 0;">
+          <table>
```

Comment:
Tables aren't meant to be used for this kind of layout so I would just do a few rows of `.resources` divs here instead.